### DISCUSSION TITLE
Only i18n certain pages, redirects to default locale for the others

### DISCUSSION THREAD

**Discussion by jeanhdev** [vercel/next.js#39645] [Help]
My website has only the home page `/` translated to english.

I have a website with content prefixed like the following

`/blog/beekeeping`

My default locale is french `fr`. 

I'd like to get crawler (and anybody really) to redirect to `/blog/beekeeping` when they try to index `/en/blog/beekeeping` as only my homepage is translated and all of my contents is only in `fr`.

I tried using `middleware.ts` with a redirect but it broke my links. Any idea on how to do it? Maybe in `next.config.js` with `redirects`?

Thank you a lot !

**Comment by fevivi** [comment]
any update?
